api/Makefile

```
@@ -97,7 +97,12 @@ django-collect-static:
 .PHONY: serve
 serve:
 	poetry run flagsmith start --reload api
-		
+
+.PHONY: run-task-processor
+serve-with-task-processor:
+	export TASK_RUN_METHOD="TASK_PROCESSOR"
+	poetry run python manage.py start --reload --bind 0.0.0.0:8001 task-processor &
+	poetry run python manage.py start --reload --bind 0.0.0.0:8000 api
```

Let's break down and compose

```suggestion
.PHONY: run-task-processor
run-task-processor:
	poetry run flagsmith start --reload --bind 0.0.0.0:8001 task-processor

.PHONY: serve-with-task-processor
serve-with-task-processor: serve run-task-processor
```

---

@khvn26 This doesn't allow the two commands to run in parallel without `-j2`.

---

True, we can change it to bash then:

```
.PHONY: run-task-processor
run-task-processor:
	poetry run flagsmith start --reload --bind 0.0.0.0:8001 task-processor

.PHONY: serve-with-task-processor
serve-with-task-processor: TASK_RUN_METHOD=TASK_PROCESSOR
serve-with-task-processor:
	make -j2 serve run-task-processor
```

This will preserve small, granular targets and keep them composable — i.e. we won't have to modify both `serve` and `serve-with-task-processor` if we change the API entrypoint. 

We could also add the `docker-up` dependency for convenience/self-documentation:

```
.PHONY: serve
serve: docker-up
	poetry run flagsmith start --reload api

.PHONY: run-task-processor
run-task-processor: docker-up
	poetry run flagsmith start --reload --bind 0.0.0.0:8001 task-processor

.PHONY: serve-with-task-processor
serve-with-task-processor: TASK_RUN_METHOD=TASK_PROCESSOR
serve-with-task-processor:
	make -j2 serve run-task-processor
```

---

Fixed with
- 70b3a27fa81576c1403425dc074cf6ee77501281
- 043a1012d85f1b7ef69cac4e9a790edfedf0ad66